src/components/organisms/rns/sell/SoldDomains.tsx

```diff
@@ -92,7 +47,7 @@ const SoldDomains: FC<{}> = () => {
       } = domainItem
       const currency = crypto[paymentToken]
 
-      const pseudoResolvedName = domainFilters?.name?.$like && `${domainFilters?.name?.$like}.rsk`
+      const pseudoResolvedName = !!name && `${name}.rsk`
```

Comment:
```suggestion
      const pseudoResolvedName = name && `${name}.rsk`
```
The `!!` is unnecessary, the end result is the same

Reply:
Well, while you are right about that the end result is the same, I thought it would be clearer if the constant held `false` instead of `''` if the string is empty.
It's not a very strong argument so, but by all means, I can change that if that didn't convince you.